### ISSUE TITLE
Add support for environment variable SHELLCHECK_OPTS

### ISSUE THREAD

**Issue by elchenberg** [hadolint/hadolint#469]
- [ ] This is a bug report
- [x] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

The [`SHELLCHECK_OPTS` environment variable](https://github.com/koalaman/shellcheck/blob/master/shellcheck.1.md#environment-variables) can be used to set default Shellcheck flags.

### Actual behavior

`SHELLCHECK_OPTS="--enable=all"`

- actual: no output
- expected: `/dev/stdin:2 SC2250: Prefer putting braces around variable references even when not strictly required.`

`SHELLCHECK_OPTS="--exclude=SC2016"`

- actual: `/dev/stdin:2 SC2016 Expressions don't expand in single quotes, use double quotes for that.`
- expected: no ouptut

### Steps to reproduce the behavior

`SHELLCHECK_OPTS="--enable=all"`

```sh
printf 'FROM alpine:3.12\nRUN echo "Path is $PATH"\n' | docker run --env SHELLCHECK_OPTS="--enable=all" --interactive --rm hadolint/hadolint:v1.18.0-15-g6664056-alpine
```

`SHELLCHECK_OPTS="--exclude=SC2016"`

```sh
printf "FROM alpine:3.12\nRUN echo 'Path is \$PATH'\n" | docker run --env SHELLCHECK_OPTS="--exclude=SC2016" --interactive --rm hadolint/hadolint:v1.18.0-15-g6664056-alpine
```

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```sh
Haskell Dockerfile Linter v1.18.0-15-g6664056
```

**Comment by lorenzo** [comment]
Have you tried ignoring the rules using the `--ignore` flag in hadolint or the config file?

**Comment by elchenberg** [comment]
Yes, using `--ignore` works fine for the `exclude` case. But I wanted to use shellcheck's `--enable` and `--severity` flags. My initial thought was that I might be able to achieve this using the environment variable. But now that some time has passed (since opening this issue) I'm not sure anymore if it was a good idea in the first place. :sweat_smile:

**Comment by tspearconquest** [comment]
I'd like to pass the --enable=all directive. Don't mind which way it is passed through; whether via env var or config file or a flag.

Is this something that would be easy to implement? I could take a stab at having an LLM do it, and would be happy to test it out locally and/or add tests against the feature, whatever it would take to get it accepted.
